src/BTConnectionManager.h

```diff
@@ -74,6 +76,7 @@ public Q_SLOTS:
     void setFakeTailMode(bool enableFakery) override;
     void setCommand(QVariantMap command) override;
     QVariantMap getCommand(const QString& command) override;
+    void setLocalBTDeviceState();
```

Comment:
Hmm... This isn't really public API, but give we might well need access to this from other classes in the future, it is perhaps not a bad idea to have the function defined as a public slot. It works, let's just do it this way :) (in other words, no changes, just me thinking out loud ;) )

Reply:
yeah, great point. It can be private probably, just used here to listen to device state changes. Can keep as is for now as you suggested that may need to use that from other places.